src/lib/hooks/swap/useWrapCallback.tsx

```diff
@@ -16,7 +16,7 @@ export enum WrapType {
   UNWRAP,
 }
 interface UseWrapCallbackReturns {
-  callback: () => Promise<ContractTransaction>
+  callback: () => Promise<ContractTransaction | void>
```

Comment:
When would this return Promise<void>?

Reply:
We dont need this actually - was leftover from old error catching logic